src/Command/CloudConfig.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\MagentoCloud\Command;
+
+use Magento\MagentoCloud\Process\CloudConfig as CloudConfigProcess;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * CLI command to friendly display the encoded cloud configuration environment variables
+ */
+class CloudConfig extends Command
```

Comment:
ECE-Tools already accumulates it's Cloud contexts, I propose to call class `ConfigShow` and command `config:show`